README.md

```
@@ -37,459 +33,17 @@ It serves as a bridge between AI tools and your analytics database,
+[development documentation]: https://cratedb-mcp.readthedocs.io/sandbox.html
+[managed on GitHub]: https://github.com/crate/cratedb-mcp
 [MCP]: https://modelcontextprotocol.io/introduction
-[MCP clients]: https://modelcontextprotocol.io/clients
-[mcp hub]: https://hub.docker.com/mcp
-[MCP support for Open WebUI]: https://docs.openwebui.com/openapi-servers/mcp/
-[MCP-to-OpenAPI proxy server (mcpo)]: https://github.com/open-webui/mcpo
-[MCP tools]: https://modelcontextprotocol.io/docs/concepts/tools
-[using Goose extensions]: https://block.github.io/goose/docs/getting-started/using-extensions/
-[uv]: https://docs.astral.sh/uv/
 
 [Bluesky]: https://bsky.app/search?q=cratedb
```

markdownlint-cli2 (0.18.1)

Link and image reference definitions should be needed
Unused link or image reference definition: "bluesky"

(MD053, link-image-reference-definitions)